### ISSUE TITLE
Problem with external WiFi cards

### ISSUE THREAD

**Issue by Process0** [cSploit/android#708]
They are not recognized..

**Comment by IwraStudios** [comment]
does your phone recognize it in iwconfig in an terminal

**Comment by Process0** [comment]
Yes, of course. I have a Nexus 5 device with Kali Nethunter Version 3.0 and in the terminal everything works fine. The interface is recognized and is able to start the promiscuous mode.
